gfw_pixetl/utils/utils.py

```diff
@@ -224,6 +225,18 @@ def union(
     return geom
 
 
+def _count_with_letters():
+    """Generate an infinite sequence of strings of uppercase letters
+    corresponding to numbers in base 26.
+
+    Taken from https://stackoverflow.com/a/29351603.
+    """
+    for size in itertools.count(1):
+        for letters in itertools.product(string.ascii_uppercase, repeat=size):
+            yield "".join(letters)
+
+
+@lru_cache(typed=False)
```

Comment:
This is cool! I didn't know about this decorator. What caused you to add it, @dmannarino? Did you notice something in testing, or did it get flagged while doing some profiling? Just curious. 🧠

Reply:
Nope, I realized that it was going to get called over and over again with the same value, and it's computationally expensive (well, A LITTLE). Possibly erring on the side of premature optimization, but innocuous enough.